packages/react/src/components/Button/index.scss

```diff
@@ -4,7 +4,9 @@
   border: thin solid transparent;
   box-sizing: border-box;
   cursor: pointer;
-  display: flex;
+  display: inline-flex;
+  // When button is rendered as link.
+  text-decoration: none;
```

Comment:
Do you check the behavior with `:visited` state ? As when a link is visited, its color changes.

Reply:
I tried setting `:visited` and the color stays the same